rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/NodesRecoveryManager.java

```diff
@@ -36,6 +36,8 @@
 
 import org.apache.log4j.Logger;
 import org.objectweb.proactive.core.node.Node;
+import org.objectweb.proactive.core.node.NodeImpl;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.objectweb.proactive.core.node.NodeImpl'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)